code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -218,10 +218,16 @@
 	force_threshold = 10
 	pressure_resistance = 50
 	mob_size = MOB_SIZE_LARGE
-	see_invisible = SEE_INVISIBLE_MINIMUM
 	see_in_dark = 8
 	var/independent = FALSE
 
+/mob/living/simple_animal/hostile/blob/blobbernaut/Login()
+	. = ..()
+	if (hud_used)
```

Comment:
ugh!

I've always hated the idea of a object var that loses usefulness after the object is initialized.

You couldn't sync changes to it without life() fuckary (or more likely, more update_sight() checks) so it becomes just a compile time to initialize time ferry, and such vars have always seemed to be hacky